src/dymaptic.GeoBlazor.Core/Components/Widgets/SearchWidget.gb.cs

```diff
@@ -269,7 +269,7 @@ public SearchWidget(
     [ArcGISProperty]
     [JsonIgnore(Condition = JsonIgnoreCondition.WhenWritingNull)]
     [JsonInclude]
-    public IReadOnlyList<object>? Results { get; protected set; }
+    public IReadOnlyList<SearchResultResponse>? Results { get; protected set; }
```

Comment:
We should try to clean up all instances of `object` as they don't play as well as concrete types when deserializing.